docs/best-practices/data-import.rst

```diff
@@ -274,12 +285,13 @@ dataset with a lot of columns and large values, it makes sense to decrease the
 ``bulk_size``. Setting ``bulk_size`` too high might consume a lot of node
 resources while a low ``bulk_size`` can increase the overhead per request.
 
+
 Compression
 ...........
 
 If you do not have your data locally to the nodes, but somewhere on the
-network, e.g. a NAS or on ``S3``, it's recommended to use ``gzip`` compressed
-files to reduce network traffic.
+network, e.g. on a NAS or on ``S3``, it's recommended to use ``gzip``
```

Comment:
```suggestion
network, e.g., on a NAS or on ``S3``, it's recommended to use ``gzip``
```

Reply:
That makes too many commas. I'm going to split it then